#ifndef BCSETTINGDISPLYMODELSTYLE_H
#define BCSETTINGDISPLYMODELSTYLE_H

#include <QDialog>

namespace Ui {
class BCSettingDisplyModelStyle;
}

class BCSettingDisplyModelStyle : public QDialog
{
    Q_OBJECT

public:
    explicit BCSettingDisplyModelStyle(QWidget *parent = 0);
    ~BCSettingDisplyModelStyle();

private slots:
    void on_m_pOKBtn_clicked();

    void on_m_pCancelBtn_clicked();

private:
    Ui::BCSettingDisplyModelStyle *ui;
};

#endif // BCSETTINGDISPLYMODELSTYLE_H
